models/base/src_base.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 sources:
   - name: atomic
     schema: "{{ var('snowplow__atomic_schema', 'atomic') }}"
-    database: "{{ var('snowplow__database', target.database) }}"
+    database: "{{ var('snowplow__database', target.database) if target.type != 'databricks' else var('snowplow__atomic_schema', 'atomic') }}"
```

Comment:
This differentiating logic made me a bit confused as based on the variable comments it is not clear why or which variables to set: 
```
  snowplow__atomic_schema: 'atomic_data_sample' # Only set if not using 'atomic' schema for Snowplow events data
   snowplow__database:  # Only set if not using target.database for Snowplow events data
```
As I was testing it, target.database is null in case of Databricks. So upon reading the variable comments it would make sense to set a database for snowplow__database variable in case of Databricks. However, I guess this could cause confusion because as I learned in Databricks schema and database are used interchangeably and database in this context does not make sense. Just thought we could elaborate on this in the docs somewhere, not necessarily in those variable comments though. However, I might be confused because I'm a 100% new to DB so not sure if we need to be that diligent, just food for thought.

Reply:
Yep, very good point. I'll elaborate in the docs that we don't need to set a database for Databricks since that concept doesn't exist in the adapter and database <> schema are interchangeable!